### PR TITLE
Add ChilliBits.Spice

### PR DESCRIPTION
**Pull request by marcauberer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50286)

### CODE REVIEW

File: manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+InstallerType: msi
+Scope: machine
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/spicelang/spice/releases/download/0.7.1/spice_x64_setup.msi
+  InstallerSha256: 5b2dd41a8811d1c898624967e08a4be7700d21c0ff9b78de78f34521582995b1
+  ProductCode: "{CE9D9BA8-0EBA-4DA8-A473-7D9C44600583}"
```
Comment (ItzLevvie) [inline manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.installer.yaml:19]:
```suggestion
  ProductCode: "{A70BEF44-08F4-405B-A361-349257A8C2FD}"
```
This must match Windows Registry.

File: manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+PackageIdentifier: ChilliBits.Spice
+PackageVersion: 0.7.1
+PackageLocale: de-DE
+Publisher: ChilliBits
+PublisherUrl: https://chillibits.com/
+PublisherSupportUrl: https://www.spicelang.com
+PrivacyUrl: https://www.chillibits.com/en/legal-notice/
+Author: Marc Auberer
+PackageName: Spice Programming Language
```
Comment (ItzLevvie) [inline manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.locale.de-DE.yaml:12]:
```suggestion
PackageName: Spice
```
This must match Control Panel.

File: manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+PackageIdentifier: ChilliBits.Spice
+PackageVersion: 0.7.1
+PackageLocale: en-US
+Publisher: ChilliBits
+PublisherUrl: https://chillibits.com/
+PublisherSupportUrl: https://www.spicelang.com
+PrivacyUrl: https://www.chillibits.com/en/legal-notice/
+Author: Marc Auberer
+PackageName: Spice Programming Language
```
Comment (ItzLevvie) [inline manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.locale.en-US.yaml:12]:
```suggestion
PackageName: Spice
```
This must match Control Panel.

**Review by ItzLevvie — APPROVED** [review]

File: manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.installer.yaml (hunk 1 above)
Comment (marcauberer) [reply manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.installer.yaml:19]: Hi @ItzLevvie. I think the bot merged the pr accidentally. I just created a [follow-up pr](https://github.com/microsoft/winget-pkgs/pull/50338) for your suggested changes.

**Pull request merged**